### PR TITLE
feat: show amount devcon round

### PR DESCRIPTION
**Pull request by ctrlc03**
Show the amount to be distributed to projects

### CODE REVIEW

**Review by kittybest — COMMENTED** [review]
Looks good to me, just left a question

File: packages/interface/src/features/results/components/ResultItem.tsx [hunk 1]
```diff
@@ -10,9 +10,19 @@ export interface IResultItemProps {
   rank: number;
   project: IRecipientWithVotes;
   medal?: EMedal;
+  totalVotes?: number;
 }
 
-export const ResultItem = ({ pollId, rank, project, medal = undefined }: IResultItemProps): JSX.Element => {
+// devcon round budget
+const budget = 250000;
```
[the remark this answers is not in the document]
Comment (ctrlc03) [reply packages/interface/src/features/results/components/ResultItem.tsx:17]: 250k USD

File: packages/interface/src/features/results/components/ResultItem.tsx [hunk 2]
```diff
@@ -39,7 +49,11 @@ export const ResultItem = ({ pollId, rank, project, medal = undefined }: IResult
 
         <div className="mx-2 flex-1">{metadata.data?.name}</div>
 
-        <div className="w-24 flex-none text-end">{`${project.votes} votes`}</div>
+        {totalVotes ? (
+          <div className="w-24 flex-none text-end">{`${((budget * project.votes) / totalVotes).toFixed(2)} $`}</div>
```
Comment (kittybest) [inline packages/interface/src/features/results/components/ResultItem.tsx:53]: then do we need to specify the unit here? like `250 USD$`, or I would be confused whether we use the real world fiat or any coins

Comment (kittybest) [reply packages/interface/src/features/results/components/ResultItem.tsx:53]: This is the last question I have, if this done, could approve it!

Comment (ctrlc03) [reply packages/interface/src/features/results/components/ResultItem.tsx:53]: I added the dollar sign, but can add usd too

Comment (kittybest) [reply packages/interface/src/features/results/components/ResultItem.tsx:53]: I saw it, maybe just me not familiar with using USD currency, seems reasonable for me

**Review by kittybest — APPROVED** [review]
Thanks!

**Pull request merged**